### DISCUSSION TITLE
[GrapheneOS] Camera picture not working

### DISCUSSION THREAD

**Discussion by Mailiax** [gsantner/markor#2117] [Q&A] [answered]
Nothing happens when trying to insert an image when tapping the camera picture button. Same problem on both GrapheneOS and LineageOS on two different devices. Anything I'm doing wrong?

Saw a similar discussion but from what I can see there is not solution provided. #2007

**Comment by gsantner** [comment, marked as answer]
> Not provided

Well, would be helpful if a Graphene user can take a look and propose a merge request. Please don't expect it to magically change unless somebody steps up.

 It works just fine on the devices I have available at last.

**Comment by Mailiax** [comment]
I see. Although it seems the problem isn't just specific to Graphene since it doesn't work on either one of my devices, including the one running Lineage. Any idea why this may be the case? Both devices are Android 13.

**Reply by gsantner** [reply]
Not so far, otherwise I at last would have answered differently or taken a look to fix it here
